### DISCUSSION TITLE
Add support for Neo plugin

### DISCUSSION THREAD

**Discussion by jamielaughton** [engram-design/FeedMe#882] [Ideas]
### Description
I have Neo fields in my entries which I want to be populated by a JSON feed pulled in through Feed Me, im using elementApi top create the JSON feed. 

The Feed Me setup runs fine and there are no errors. My issue is that I'm un-able to map the Neo Fields when it comes to the configuration screen (where you select the Feed Element from your fields to map with the JSON fields), because I am only seeing the top-level of the Neo fields.

Any help would be greatly appreciated 

Cheers,
Jamie
### Additional info

- Plugin version: 2.0.11
- Craft version: 2.6.3013

**Comment by engram-design** [comment]
@jamielaughton Unfortunately, Neo support isn't ready for Feed Me, which would explain why you're seeing these specific results. We can't provide an ETA on this unfortunately, but it'll be done (along with other populate third-party fields and elements) before the end of beta - likely in 1-2 months.

I should also mention - this would be for Feed Me 3. There are no plans to implement this functionality for Feed Me 2, which isn't going to implement new features - just fixes.

**Comment by jamielaughton** [comment]
Thanks for the reply, 

Good to hear it will be supported soon, I'll keep an eye out for the updates!

**Comment by reganlawton** [comment]
👍 for this feature. I'll trade a kidney

**Comment by davidhellmann** [comment]
We need it too :|

**Comment by paulalden** [comment]
Any progress on this, be an absolute life saver.

**Comment by tekstrand** [comment]
👍  would like to see this

**Comment by davidwebca** [comment]
I ran into this issue. Tried to export / import with feedme and I wondered why my neo fields wouldn't show as single items that I could map. I'm even using a combo of Neo + Super table 😢

**Comment by ttempleton** [comment]
Feed Me support now included in Neo 3.5.0 - see the [usage document](https://github.com/spicywebau/craft-neo/blob/3.5.0/docs/feed-me.md) for details.
